client/state/teams/actions.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Internal dependencies
+ */
+import { TEAMS_REQUEST } from 'calypso/state/teams/action-types';
+
+import 'calypso/state/data-layer/wpcom/read/teams';
+
+import 'calypso/state/teams/init';
```

Comment:
Perhaps it is a good time changing this too? It will require a wpcom change also to `read/teams`

Reply:
This line is unrelated to wpcom or anything outside of Calypso. It's an artifact of how we handle [modularised state](https://github.com/Automattic/wp-calypso/blob/trunk/docs/modularized-state.md).

Reply:
Sorry, I meant `import 'calypso/state/data-layer/wpcom/read/teams';` on line (6).

Reply:
Oh, I see! Yes, the paths could certainly be standardised at some point 👍